openapi.yaml

```diff
@@ -2087,7 +2087,7 @@ components:
         The value can be repeated to retrieve multiple matching values (OR condition).
     FilterQueue:
       in: query
-      name: Queue
+      name: Qqueue
```

Comment:
Here I think there is a typo?

Reply:
Fixed. Thanks.
https://github.com/apache/airflow/pull/8721/commits/084bed4fbb0257156dae9407b00478ac2f2136ee